src/SFA.DAS.ApprenticePortal.Web/Pages/Home.cshtml.cs

```diff
@@ -2,24 +2,44 @@
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using SFA.DAS.ApprenticePortal.SharedUi.Home;
 using SFA.DAS.ApprenticePortal.Web.Services;
-using SFA.DAS.ApprenticePortal.Web.Services.OuterApi;
 using System.Threading.Tasks;
+using SFA.DAS.ApprenticePortal.Web.Models;
 
 namespace SFA.DAS.ApprenticePortal.Web.Pages
 {
     [Authorize]
     public class HomeModel : PageModel
     {
-        private readonly IOuterApiClient _client;
+        private readonly ApprenticeshipService _apprenticeshipService;
         private readonly AuthenticatedUser _user;
         private readonly NotificationAccessor _notifications;
 
-        public string CourseName { get; set; } = null!;
-        public bool ApprenticeshipComplete { get; set; } = false;
+        public ApprenticeshipModel? CurrentApprenticeship { get; set; }
 
-        public HomeModel(IOuterApiClient client, AuthenticatedUser user, NotificationAccessor notifications)
+        public ApprenticeshipStatus Status

```

Comment:
Would it make sense to extract this to an extension method?

Reply:
yes. agree